aspnet/security/authentication/cookieauthentication.rst

```diff
@@ -0,0 +1,166 @@
+* **CookiePath** - this can be used to isolate applications running on the same host name. By default any cookies generated will have a path of ``\`` which makes them available to any request path on your website. If you have an app running in ``/app1`` and want to limit the cookies issued to just be sent to that application then you should set the ``CookiePath`` property to ``/app1``. The cookie will now only be available to requests to ``//app1`` or anything underneath it.
+* **CookieSecure** - a flag indicating if the cookie created should be limited to HTTPS, HTTP or HTTPS, or the same protocol as the request. This defaults to ``SameAsRequest``. 
+* **ExpireTimeSpan** - the ``TimeSpan`` after which the cookie will expire. This is added to the current date and time to create the expiry date for the cookie.
+* **SlidingExpiration** - a flag indicating if the cookie expiration date will be reset when the more than half of the ``ExpireTimeSpan`` interval has passed. The new expiry date will be moved forward to be the current date plus the ``ExpireTimespan``. An :ref:`absolute expiry time <security-authentication-absolute-expiry>` can be set by using the ``AuthenticationProperties`` class when calling ``SignInAsync``. An absolute expiry can improve the security of your application by limiting the amount of time for which the authentication cookie is valid.
+
+Non-persistent cookies and absolute expiry times
+------------------------------------------------
+
+You may want to make the cookie expire when the browser is closed, or be remembered over browser sessions. You may also want an absolute expiry to the identity and the cookie transporting it. You can do these things by using the ``AuthenticationProperties`` parameter on the ``HttpContext.Authentication.SignInAsync`` method called when :ref:`signing in an identity and creating the cookie<security-authentication-cookie-middleware-creating-a-cookie>`. The ``AuthenticationProperties`` class is in the ``Microsoft.AspNet.Http.Authentication`` namespace.
+
+For example;
+
+.. code-block:: c#
+
+ await HttpContext.Authentication.SignInAsync(
+     "MyCookieMiddlewareInstance", 
+     principal,
+     new AuthenticationProperties
+     {
+         IsPersistent = false                        
+     });
+
+This code snippet will create an identity and corresponding cookie which will be cleared when the browser closes. This ignores any sliding expiration settings previously configured via :ref:`cookie options <security-authentication-cookie-options>`.
+
+.. _security-authentication-absolute-expiry:
+
+.. code-block:: c#
+
+ await HttpContext.Authentication.SignInAsync(
+     "MyCookieMiddlewareInstance", 
+     principal,
+     new AuthenticationProperties
+     {
+         ExpiresUtc = DateTime.UtcNow.AddMinutes(20)
```

Comment:
Damn, removing `IsPersistent` won't work. You need to specify `IsPersistent = true` if you want to avoid issuing a session cookie :smile: